src/components/draw/feature-edit-style-point.vue

```diff
@@ -4,4 +4,5 @@
   <slot name="color"></slot>
   <slot name="symbole"></slot>
   <slot name="size"></slot>
+  <slot name="angle"></slot>
```

Comment:
Does the point style edition really need the angle?

Reply:
you are right, only the label does